flowzone.yml

```diff
@@ -1605,9 +1611,6 @@ jobs:
       - project_types
       - versioned_source
       - event_types
-    if: |
-      needs.project_types.outputs.has_readme == 'true' &&
-      inputs.cloudflare_website != ''
```

Comment:
We still need to check that `inputs.cloudflare_website` is not empty here, because both docusaurus and custom still push to CF.

Reply:
Okay yeah that's a good catch